### ISSUE TITLE
Update dependencies: spring-security-oauth2 e hazelcast*

### ISSUE THREAD

**Issue by phouverneyuff** [jhipster/generator-jhipster#1616]
The current version to oauth in project is 2.0.1.RELEASE, but exist the new version 2.0.7.RELEASE.
Link to see all versions:
http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22org.springframework.security.oauth%22%20AND%20a%3A%22spring-security-oauth2%22

The same occur with hazelcast. In project is 3.4, but 3.4.3 is avaliable.
http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22com.hazelcast%22%20AND%20a%3A%22hazelcast-spring%22

**Comment by phouverneyuff** [comment]
I changed in my project, but on update Hazelcast to 3.4.3, the build fails in class WebConfigurer.
Until the version 3.4.2 the build works.
"WebConfigurer.java:[74,13] method does not override or implement a method from a supertype"

I found in release notes that they changed the way to create instances.

http://docs.hazelcast.org/docs/3.4/manual/html-single/hazelcast-documentation.html#release-notes

https://github.com/hazelcast/hazelcast/issues/3781

"The Hazelcast class is not an instance of HazelcastInstance. For the same reasons a HazelcastClient should not be a HazelcastInstance. It should be a 'factory' and provide the ability to shut them down."

```JAVA
 WebFilter webFilter = new WebFilter() {
     @Override
     protected HazelcastInstance getInstance(Properties properties) throws ServletException {
          return CacheConfiguration.getHazelcastInstance();
     }
};
```
